src/Website/Silk.NET.Statiq/Utilities.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Html;
+using Statiq.Common;
+using Statiq.Razor;
+
+namespace Silk.NET.Statiq
+{
+    public abstract class SilkPage : StatiqRazorPage<IDocument>
+    {
+        /// <summary>
+        /// Converts a path which is relative to the input path to a URL which is relative to the current page's
+        /// containing directory.
+        /// </summary>
+        /// <param name="path">The path.</param>
+        /// <returns>The URL.</returns>
+        public string InputUrl(string path)
```

Comment:
Why is this needed? Seems like it's only used for the JS/CSS files, which should be absolute to the root anyways, which is just an HTTP feature. Just start the path of a referenced file with a leading '/' and then it'll be an absolute path.

Reply:
This is used for all URLs. I don't want to have it use `/something`, I want to ensure all URLs are relative mainly to keep the site operable outside of a server environment as well as for future enhancements such as pull request publishing (i.e. we could have `https://dotnet.github.io/Silk.NET/pr/612` so we can view the rendered webpages, something I can see being useful when we have tutorial PRs)